pipelines/broad/dna_seq/germline/single_sample/exome/ExomeGermlineSingleSample.changelog.md

```diff
@@ -1,3 +1,8 @@
+# 3.0.7
+2022-04-15 (Date of Last Commit)
+
+* Updated task SortSam in BamProcessing.wdl, this update has no effect on this pipeline
```

Comment:
Minor nit - I think BamProcessing is used in all of these affected pipelines, so you might change this comment to say something like 'updated task SortSam in BamProcessing.wdl to take an optional memory_multiplier.
(but I could be wrong that it's not used in the affected pipelines).